manta-crypto/src/algebra.rs

```diff
@@ -56,15 +67,15 @@ pub trait Group<COM = ()> {
 )]
 #[derive(derivative::Derivative)]
 #[derivative(Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
-pub struct DiffieHellman<G, COM = ()> {
+pub struct DiffieHellman<S, G> {
```

Comment:
If we mention it here then we need to provide the `COM` argument which is just more verbose. It's better to avoid the `COM` argument on structures and keep them on `trait` definitions and `trait` implementations.